lua/dap/session.lua

```diff
@@ -842,7 +842,12 @@ end
 
 
 function Session:close()
-  vim.fn.sign_unplace(ns_pos)
+  -- can't always be called safely, returns the below with
+  -- jbyuki/one-small-step-for-vimkind
+  -- E5560: vimL function must not be called in a lua loop callback
+  vim.schedule(function()
+    pcall(vim.fn.sign_unplace, ns_pos)
```

Comment:
Why is the pcall also necessary?

Reply:
Technically it’s not necessary, just wanted to be on the safe side, can be removed, up to you.

Reply:
Rbased the commit, `pcall` removed.